module/VuFind/src/VuFind/ChannelProvider/ListItemsFactory.php

```diff
@@ -71,7 +71,7 @@ public function __invoke(
         return new $requestedName(
             $container->get(\VuFind\Db\Table\PluginManager::class)->get('UserList'),
             $container
-                ->get(\VuFind\Db\Table\PluginManager::class)->get('ResourceTags'),
+                ->get(\VuFind\Db\Service\PluginManager::class)->get('tag'),
```

Comment:
Would it be better to use the fully-qualified class name here instead of the alias? That's what we seem to be doing most of the time, and it's good to be consistent.